internal/store/builder.go

```diff
@@ -319,14 +326,27 @@ func (b *Builder) buildStore(
 	expectedType interface{},
 	listWatchFunc func(kubeClient clientset.Interface, ns string) cache.ListerWatcher,
 ) cache.Store {
-	filteredMetricFamilies := generator.FilterMetricFamilies(b.whiteBlackList, metricFamilies)
+	filteredMetricFamilies := generator.FilterMetricFamilies(b.whiteBlackList, b.enableAggregations, metricFamilies)
```

Comment:
It's hard to follow all the dependencies unless I pull the branch, so I'll just ask here: does FilterMetricFamilies take into account the "--metric-denylist" ?